src/components/common/AppBreadcrumbs.tsx

```diff
@@ -102,20 +117,64 @@ const AppBreadcrumbs = (props: Props) => {
     Boolean,
   );
   return (
-    <Breadcrumbs className={classes.root} separator={<NavigateNextIcon fontSize="small" />} aria-label="breadcrumb">
-      {crumbs.map((crumb, i) => (
-        <Crumb
-          key={crumb.name}
-          active={crumbs.length - 1 === i}
-          name={crumb.name}
-          href={crumb.href}
-          Icon={crumb.Icon}
-        />
-      ))}
-    </Breadcrumbs>
+    <Stack className={classes.root} direction="row" spacing={1}>
+      {viewer.relatedOwners && viewer.relatedOwners.length === 1 ? null : <AccountsButton viewer={viewer} />}
+      <Breadcrumbs
+        className={classes.breadcrumbs}
+        separator={<NavigateNextIcon fontSize="small" />}
+        aria-label="breadcrumb"
+      >
+        {crumbs.map((crumb, i) => (
+          <Crumb
+            key={crumb.name}
+            active={crumbs.length - 1 === i}
+            name={crumb.name}
+            href={crumb.href}
+            Icon={crumb.Icon}
+          />
+        ))}
+      </Breadcrumbs>
+    </Stack>
   );
 };
 
+interface AccountsButtonProps extends WithStyles<typeof styles> {
+  viewer: AppBreadcrumbs_viewer;
+}
+
+const AccountsButton = styled(({ viewer, classes }: AccountsButtonProps) => {
+  const navigate = useNavigate();
+
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(e.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleMenuItemClick = (e, name) => {
+    setAnchorEl(null);
+    navigateHelper(navigate, e, '/github/' + name);
+  };
+
+  return (
+    <>
+      <Button variant="contained" onClick={handleClick}>
+        Accounts
+        <ArrowDropDownIcon />
+      </Button>
+      <Menu anchorEl={anchorEl} open={open} onClose={handleClose}>
+        {viewer.relatedOwners.map(viewer => {
+          return <MenuItem onClick={e => handleMenuItemClick(e, viewer.name)}>{viewer.name}</MenuItem>;
+        })}
+      </Menu>
```

Comment:
Have you considered using a [`Select` component](https://mui.com/material-ui/react-select/#basic-select) so you don't need to manage the popup?

Reply:
I’ve tried but I have doubts about Select. I’m not sure that we need logic here with displayed selected value. I think it’s more nice when you see selected value in the root of Breadcrumbs without duplication. What do you think?